client/sections.js

```diff
@@ -108,6 +108,13 @@ const sections = [
 		module: 'calypso/my-sites/plugins',
 		group: 'sites',
 	},
+	{
+		// TODO: Move this out to its own root folder
+		name: 'marketplace',
+		paths: [ '/marketplace' ],
```

Comment:
I'd like to double-check and verify that this is the URL structure that we want to go with. Once this is introduced and used for something, it could cause some pain to change 😉

Reply:
We currently have no elaborate design, discussion or concensus on the route structure, can you tell me what kind of risk it is and why it would be a potential pain point to change the route structure in the future? 🤔

Reply:
The idea is that if we introduce this to the public, and eventually we decide to rename it, we'll have to create redirects, potentially update any links and documentation that references the old links as well. I don't have reservations on the naming specifically, just wanted to get your confirmation that it's been agreed both from a product and marketing perspective that `http://wordpress.com/marketplace/*` will be the final destination of this new section.

Reply:
@paulbonahora @JanaMW27 just a small heads up here. The route structure for our marketplace stuff would be 
`http://wordpress.com/marketplace/*`. I know we haven't discussed this but it might be bit of a brittle change so looping you in for thoughts.